tests/lib/ocpp/v201/test_smart_charging_handler.cpp

```diff
@@ -0,0 +1,276 @@
+#include "date/tz.h"
+#include "ocpp/v201/ctrlr_component_variables.hpp"
+#include "ocpp/v201/device_model_storage_sqlite.hpp"
+#include <boost/uuid/uuid_generators.hpp>
+#include <boost/uuid/uuid_io.hpp>
+#include <filesystem>
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include <memory>
+namespace fs = std::filesystem;
+
+#include <evse_security_mock.hpp>
+#include <ocpp/common/call_types.hpp>
+#include <ocpp/v201/enums.hpp>
+#include <ocpp/v201/evse.hpp>
+#include <ocpp/v201/smart_charging.hpp>
+#include <optional>
+
+#include <sstream>
+
+namespace ocpp {
+namespace v201 {
+
+class DeviceModelStorageMock : public DeviceModelStorage {
```

Comment:
Might be smart to put these mocks in their own include files so they can be reused?